cityofaustin/atd-vz-data
# Add new "Notes" section to Crash Details page

_Note: This is @roseeichelmann's PR, I just helped get this first PR set up. I will be reviewing though Github doesn't let the PR author assign themself to review_

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/9496

## Testing
**URL to test:** 

This PR can be tested locally.

https://localhost:3000/#/crashes/17727381

_Note from Mateo: 
You can also review the PR as a Netlify deploy preview - https://deploy-preview-1073--atd-vze-staging.netlify.app/#/crashes/17727381_

**Steps to test:**

1. Click on the Crashes tab.
2. Go to crash ID 17727381
3. Notes component should be visible with test data.
4. Go to any other crash ID, notes component should display but be empty.


---
#### Ship list
